src/pyhf/schema/loader.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+import sys
+import json
+import pyhf.exceptions
+from pyhf.schema import variables
+
+# importlib.resources.as_file wasn't added until Python 3.9?
+if sys.version_info >= (3, 9):
+    from importlib import resources
+else:
+    import importlib_resources as resources
```

Comment:
In a follow up PR we should see if it would be worth making this

https://github.com/scikit-hep/pyhf/blob/1884c6c08bc5b629724dd9ef3c60ec4386353b94/.github/workflows/ci.yml#L70

be for both the oldest and newest Python supported (so Python 3.7 and Python 3.10) so that we cover these sorts of things in our reports.

Reply:
This is now Issue #1816.